### PR TITLE
footer config update

### PR DESCRIPTION
**Pull request by andrii-moskalenko**

### CODE REVIEW

**Review by Poiar — CHANGES_REQUESTED** [review]

**Comment by andrii-moskalenko** [comment]
@Poiar I've added settings menu option 'Page footer'

File: MM/5.10.0/config/media_manager_5/labels/Settings.dcl [hunk 1]
```diff
@@ -206,6 +206,55 @@ resource configservice_label settings_splashscreen_settings_upload_background_as
...
+  key = 'SETTINGS_PAGEFOOTER_SETTINGS_DESCRIPTION'
+  group = 'Settings'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = '{languageId, select, 1 {Danish f} 3 {English f} other {F}}ooter text'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = '{languageId, select, 1 {Dansk } 3 {Engelsk } other {}}Footer text'
```
Comment (Poiar) [inline MM/5.10.0/config/media_manager_5/labels/Settings.dcl:219]:
```suggestion
      default_translation = '{languageId, select, 1 {Dansk footer} 3 {Engelsk footer} other {Footer}} text'
```

File: MM/5.10.0/config/media_manager_5/labels/Settings.dcl [hunk 2]
```diff
@@ -206,6 +206,55 @@ resource configservice_label settings_splashscreen_settings_upload_background_as
   ]
 }
 
+resource configservice_label settings_pagefooter_settings_description {
+  key = 'SETTINGS_PAGEFOOTER_SETTINGS_DESCRIPTION'
+  group = 'Settings'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = '{languageId, select, 1 {Danish f} 3 {English f} other {F}}ooter text'
```
Comment (Poiar) [inline MM/5.10.0/config/media_manager_5/labels/Settings.dcl:215]:
```suggestion
      default_translation = '{languageId, select, 1 {Danish footer} 3 {English footer} other {Footer}} text'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/5.10.0/config/media_manager_5/labels/Settings.dcl [hunk 3]
```diff
@@ -334,6 +383,22 @@ resource configservice_label settings_settings_root_labels {
...
+  key = 'SETTINGS_SETTINGS_ROOT_PAGEFOOTER'
+  group = 'Settings'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Page footer'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Page footer'
```
Comment (Poiar) [inline MM/5.10.0/config/media_manager_5/labels/Settings.dcl:396]:
```suggestion
      default_translation = 'Side-footer'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

**Review by Poiar — APPROVED** [review]

**Pull request merged**